downstream/modules/eda/proc-eda-set-up-new-project.adoc

```diff
@@ -0,0 +1,37 @@
+[id="proc-eda-set-up-new-project"]
+
+= Setting up a new project
+
+[role="_abstract"]
+
+Projects are a logical collection of rulebooks.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have set up a credential, if necessary. 
+For more information, refer to the link:https://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html[Credentials]
+section of the {ControllerNameStart} documentation.
+* You have an existing repository containing rulebooks that are integrated with playbooks contained in a repository to be used by Automation Controller.
+
+== Procedure
+
+. Log in to the {EDAcontroller} Dashboard.
+. From the navigation panel, select menu:Projects[Create project].
+. Insert the following:
++
+Name::
+Description:: This field is optional.
+SCM type:: Git is the only SCM type available for use.
+SCM URL:: This is a URL to a remote archive, such as GitHub Release or a build artifact stored in Artifactory that unpacks it into the project path for use.
```

Comment:
I think this is notcorrect. It must be a http git repository such a github or gitlab. It would support ssh urls only if are unauthenticated. (authentication is only supported for http)

In behind, it runs `git clone` command, so if the url points to an artifact, it will fail.

Reply:
As above, this is what it looks like in the mockups.

Reply:
@Alex-Izquierdo are you saying the help text for the SCM URL is wrong? I got those from Ansible docs somewhere and Joe and Ben reviewed help text with me. Let me know if it should be adjusted.

Reply:
Indeed. Only http[s] protocol is currently supported. It doesn't support archives. UI also validates only http urls. @benthomasson can confirm it.
I suggest:
```suggestion
SCM URL:: HTTP[S] protocol address of a git repository
```
Refs:
- https://github.com/ansible/ansible-ui/blob/a5b93e9b01b3be2c88bcc4fbd898c3fecfdf6838/frontend/eda/Resources/projects/EditProject.tsx#L55
- https://github.com/ansible/aap-eda/blob/main/src/aap_eda/services/project/git.py#L95

Reply:
Yes Alex is correct.